### ISSUE TITLE
options is undefined, when using as a module

### ISSUE THREAD

**Issue by ffoodd** [SphinxKnight/compat-tester#14]
Trying to use `compat.htmlFileAnalyzer()` (the module way) I'm facing an error:
```javascript
if((options.contrib === "true" || options.contrib === "all" ) && versionAddedElem === true){
           ^
TypeError: Cannot read property 'contrib' of undefined
```

But as far as I understand, there's no way to provide options when using compat-tester as a module, is there?

**Comment by ffoodd** [comment]
Or is it related to #11?

**Comment by SphinxKnight** [comment]
@ffoodd Yes, there isn't any options (yet) for the module.
I'll dig into this tonight if that can be easily improved :) 

Thanks for the report.

**Comment by SphinxKnight** [comment]
Well, adding an options argument to be used is an improvement.
Dealing with this error is a fix :/

**Comment by ffoodd** [comment]
FWIW I don't care about passing options, I'd just want to be able to run it as a module for now :)

**Comment by SphinxKnight** [comment]
@ffoodd now fixed :) with d2ec27b 
e659648 added the "options" to module use case too

I also realized that, used as a module, the report wasn't really helpful as it still was just a console.log side effect (u_u).

So you can now grab the 0.2.10 from npm to do something like:

    const compatTester = require("compat-tester");
    const browserScope = {"ie":"1", "firefox":"1", "qq_android":"1"};
    const options = {"contrib": "true"};
    const callbackReport = console.log;
    let report = compatTester.htmlStringAnalyzer('<img srcset="a">a</img>', browserScope, callbackReport, options);

I hope this helps. Feel free to raise more issues :)
I also need to think a bit more about what the module API should handle (streams, etc.) and how it should behave (sync/callbacks/Promises/async/etc.) so please still consider this as alpha :)

**Comment by ffoodd** [comment]
Thanks, on my way to try :)
I'm working on [a tool that simply gathers various testing tools](https://github.com/ffoodd/staats/), you may have a look at them if it can help in any way!
